libcontainer/configs/config.go

```diff
@@ -244,11 +244,13 @@ func (hooks Hooks) MarshalJSON() ([]byte, error) {
 
 // HookState is the payload provided to a hook on execution.
 type HookState struct {
-	Version    string `json:"ociVersion"`
-	ID         string `json:"id"`
-	Pid        int    `json:"pid"`
-	Root       string `json:"root"`
-	BundlePath string `json:"bundlePath"`
+	Version     string            `json:"version"`
+	ID          string            `json:"id"`
+	Status      string            `json:"status"`
+	Pid         int               `json:"pid"`
+	Root        string            `json:"root"`
+	BundlePath  string            `json:"bundlePath"`
+	Annotations map[string]string `json:"annotations"`
```

Comment:
Since you've added fields, can you please update the code to actually fill them? And also add some tests.